systemvm/js/jquery.flot.navigate.js

```diff
@@ -0,0 +1,336 @@
+/*
```

Comment:
@vedulasantosh license?

Reply:
@rhtyd Updated the license. Added the Apache license on top of Jquery license. Is it acceptable?

Reply:
@rhtyd can you review this license

Reply:
@vedulasantosh if the license used in this specific library are same as those used by other jquery files then I suppose we should be okay, however can you start a discussion on dev@?

Reply:
@rhtyd for all other jquery files there is no apache license because they are placed in exclude tag in pom.xml. while pushing this jquery.flot.navigate.js checks are failing at license. That is why i added apache license to this file.

pom.xml
`<exclude>ui/lib/flot/jquery.flot.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.navigate.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.pie.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.resize.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.selection.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.stack.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.symbol.js</exclude>
              <exclude>ui/lib/flot/jquery.flot.threshold.js</exclude>
              <exclude>ui/lib/jquery-ui/css/jquery-ui.css</exclude>
              <exclude>ui/lib/jquery-ui/index.html</exclude>
              <exclude>ui/lib/jquery-ui/js/jquery-ui.js</exclude>
              <exclude>ui/lib/jquery.cookies.js</exclude>
              <exclude>ui/lib/jquery.easing.js</exclude>
              <exclude>ui/lib/jquery.js</exclude>
              <exclude>ui/lib/jquery.md5.js</exclude>
              <exclude>ui/lib/jquery.validate.js</exclude>
              <exclude>ui/lib/jquery.tokeninput.js</exclude>
              <exclude>ui/lib/qunit/qunit.css</exclude>
              <exclude>ui/lib/qunit/qunit.js</exclude>
              <exclude>ui/lib/reset.css</exclude>
              <exclude>ui/lib/require.js</exclude>
              <exclude>utils/testsmallfileinactive</exclude>
              <exclude>systemvm/conf/agent.properties</exclude>
              <exclude>systemvm/conf/environment.properties</exclude>
              <exclude>systemvm/js/jquery.js</exclude>`

systemvm/js/jquery.js

`/*
 * jQuery JavaScript Library v1.3.2
 * http://jquery.com/
 *
 * Copyright (c) 2009 John Resig
 * Dual licensed under the MIT and GPL licenses.
 * http://docs.jquery.com/License
 *
 * Date: 2009-02-19 17:34:21 -0500 (Thu, 19 Feb 2009)
 * Revision: 6246
 */
(function(){var l=this,g,y=l.jQuery,p=l.$,o=l.jQuery=l.$=function(E,F){return new o.fn.init(E,F)},D=/^[^<]*(<(.|\s)+>)[^>]*$|^#([\w-]+)$/,f=/^.[^:#\[\.,]*$/;o.fn=o.prototype={init:function(E,H){E=E||document;if(E.nodeType)`

Reply:
@rhtyd can i place exclude tag for this jquery.flot.navigate.js file?

Reply:
@vedulasantosh if the existing jquery library files have the same license as the new one you're adding, then yes you may exclude this file from RAT checks, this makes sense.

Reply:
@rhtyd Made discussion on dev@. Got reply to add exclude tag for the jquery file.so I added the exclude tag in pom.xml for that file.

Reply:
@rhtyd can you please review these changes.

Reply:
@rhtyd can you please review these changes. need to merge this PR.

Reply:
@vedulasantosh sure, add this new library dependency in maven exclusion list, however, then remove the Apache license unless you authored this library.